app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
```

Comment:
- [ ] it has validations for the presence of each of the fields
- [ ] **[OPTIONAL]** We can add validation to email format.